README.md

```diff
@@ -136,17 +181,17 @@ function MyChart() {
 This will display a custom error message with a title and body text, as shown in the following image:
 
 <p align="center">
-  <img src="https://storybook.propeldata.com/images/error.png">
+  <img src="https://storybook.propeldata.com/images/error.png"/>
 </p>
 
 By customizing your Propel UI Kit components in this way, you can create unique, user-friendly data visualizations that fit seamlessly into your front-end applications.
 
-Made with :purple_heart: and [Chart.js](https://www.chartjs.org/). Powered by Propel! :rocket:
+Made with :purple_heart: and [Chart.js](https://www.chartjs.org/). Powered by [Propel](https://www.propeldata.com)! 🚀
```

Comment:
Lose this? It's currently nested under `### Errors` which feels off?

Reply:
Moving to the bottom